osmosis-1/osmosis.zone_assets.json

```diff
@@ -2593,6 +2593,12 @@
       "base_denom": "factory/osmo1279xudevmf5cw83vkhglct7jededp86k90k2le/RAPTR",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo1rckme96ptawr4zwexxj5g5gej9s2dmud8r2t9j0k0prn5mch5g4snzzwjv/sail",
+      "twitter_URL": "https://twitter.com/Sail_DAO_",
+      "osmosis_verified": true
```

Comment:
please also add osmosis_unlisted so we can first validate that everything looks OK.